src/helpers/sendNotifications.js

```diff
@@ -0,0 +1,228 @@
+import Pusher from 'pusher';
+import dotenv from 'dotenv';
+import models from '../db/models';
+import EmailNotificationAPI from './EmailNotificationAPI';
+
+dotenv.config();
+
+const {
+  Art, Comment, Following, User
+} = models;
+
+const pusher = new Pusher({
+  appId: process.env.PUSHER_APPID,
+  key: process.env.PUSHER_KEY,
+  secret: process.env.PUSHER_SECRET,
+  cluster: process.env.PUSHER_CLUSTER,
+  encrypted: true
+});
+
+/** SendNotifications Controller Class */
+class SendNotifications {
+  /**
+   * @constructor
+   * @param {Object} payloadObject - Details of the notification.
+   */
+  constructor(payloadObject) {
+    this.payloadObject = payloadObject;
+  }
+
+  /**
+   * @Represents a newArticleNotification
+   * @param {Object} articleDetailsObject - The details of the article.
+   * @return {Object|string} success string OR Error message .
+   */
+  static async newArticleNotification(articleDetailsObject) {
+    const {
+      artId, artistId, slugifiedTitle, artTitle, artFeaturedImg, artDescription
+    } = articleDetailsObject;
+
+    const artistUsername = await User.findOne({
+      where: { id: artistId },
+      attributes: ['username'],
+    });
+
+    const followers = await Following.findAll({
+      where: { userId: artistId },
+      include: [
+        {
+          model: User,
+          attributes: ['id', 'email'],
+        }
+      ],
+      attributes: [],
+    });
+
+    const mailList = [];
+
+    followers.forEach((follower) => {
+      const { id, email } = follower.User;
+      mailList.push(email);
+
+      pusher.trigger('new-article', `article-${artId}-${id}-event`, {
```

Comment:
What's this guy used for?

Reply:
This creates a new unique notification event for each follower when there's a new art created by the artist, using the [pusher](https://pusher.com/docs/javascript_quick_start) event library.